python/tank/platform/engine.py

```diff
@@ -192,6 +194,10 @@ def __init__(self, tk, context, engine_instance_name, env):
         qt_abstraction.QtCore = qt.QtCore
         qt_abstraction.QtGui = qt.QtGui
 
+        # load the fonts. this will work if there is a QApplication instance
+        # available.
+        self._ensure_core_fonts_loaded()
```

Comment:
![screen shot 2016-11-14 at 11 32 05 am](https://cloud.githubusercontent.com/assets/2604646/20273094/0a482d4e-aa5e-11e6-9891-c14f46263569.png)

... and then the instance methods:

![screen shot 2016-11-14 at 11 32 18 am](https://cloud.githubusercontent.com/assets/2604646/20273093/0a3acc3a-aa5e-11e6-83ef-2e8817be7a9e.png)